### PR TITLE
fix tests to pass against the latest tiledb dev (v1.6.0)

### PR DESCRIPTION
**Pull request by jakebolewski**

### CODE REVIEW

**Comment by jakebolewski** [comment]
Supersedes #169

**Comment by ihnorton** [comment]
Looks like there's some overlap with #169, do you want to cherry-pick/close that? Open pending tests on windows which won't pass yet due to the overlapped-context issue.

**Comment by ihnorton** [comment]
Sorry, missed your comment as I was typing mine.

File: tiledb/libtiledb.pyx [hunk 1]
```diff
@@ -4558,48 +4558,29 @@ cdef class DenseArray(Array):
...
-            rc = tiledb_query_set_buffer(ctx_ptr, query_ptr, attr_name_ptr, buff_ptr, &buff_size)
-            if rc != TILEDB_OK:
-                _raise_ctx_err(ctx_ptr, rc)
-            with nogil:
-                rc = tiledb_query_submit(ctx_ptr, query_ptr)
-            if rc != TILEDB_OK:
-                _raise_ctx_err(ctx_ptr, rc)
-        finally:
-            tiledb_query_free(&query_ptr)
-        return out
```
Comment (ihnorton) [inline tiledb/libtiledb.pyx]: 👍 oof

**Pull request merged**